python_modules/libraries/dagster-dbt/dagster_dbt/cloud/types.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+from typing import Any, Dict, Optional
+
+from dagster import check, usable_as_dagster_type
+from dateutil.parser import isoparse
+
+from ..types import DbtOutput
+
+
+@usable_as_dagster_type
+class DbtCloudOutput(DbtOutput):
```

Comment:
I don't love these output objects, because they're not resilient to changes in the underlying dbt output, but I suppose we should keep it for consistency?